### PR TITLE
Custom sc + va fix for VPC Block

### PR DESCRIPTION
**Pull request by prankulmahajan**

### CODE REVIEW

File: config-templates/ibm/ibm-vpc-block-csi-driver/4.3.0/metadata.json [hunk 1]
```diff
@@ -1,50 +1,46 @@
...
+      "value": "ibm"
+    },
+    {
+      "name": "volume-type",
+      "value": "block"
+    },
+    {
+      "name": "offering-name",
+      "value": "ibm-vpc-block-csi-driver"
+    },
```
Comment (nkkashyap) [inline config-templates/ibm/ibm-vpc-block-csi-driver/4.3.0/metadata.json:19]: Pls do not remove `provider-logo`
```
      {
        "name": "provider-logo",
        "value": "logo/ibm-vpc-block-csi-driver.svg"
      },
```

**Review by nkkashyap — CHANGES_REQUESTED** [review]

File: config-templates/ibm/ibm-vpc-block-csi-driver/4.3.0/metadata.json (hunk 1 above)
Comment (prankulmahajan) [reply config-templates/ibm/ibm-vpc-block-csi-driver/4.3.0/metadata.json:23]: @nkkashyap Updated

**Review by nkkashyap — APPROVED** [review]
lgtm

**Review by nkkashyap — APPROVED** [review]
lgtm

**Review by mssachan — APPROVED** [review]
LGTM

File: config-templates/ibm/ibm-vpc-block-csi-driver/4.3.0/storage-class-parameters.json [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        "name": "volumeBindingMode",
+        "templateKeyName": [
+            "volumeBindingMode"
+        ],
+        "default": "WaitForFirstConsumer",
+        "required": "false",
+        "category": "config"
+    },
+    {
+        "description": "Enter pre-defined profile (10iops-tier, 5iops-tier, 3iops-tier), or enter 'custom' to use a custom IOPs value. To find supported storage sizes for a specific profile, see https://cloud.ibm.com/docs/vpc?topic=vpc-block-storage-profiles. Any PVC that uses this storage class must specify a size value that is within this range.",
```
Comment (mssachan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/4.3.0/storage-class-parameters.json:35]: @prankulmahajan Please check last sentence of description. I hope this does not confuse user
`Any PVC that uses this storage class must specify a size value that is within this range`

Comment (prankulmahajan) [reply config-templates/ibm/ibm-vpc-block-csi-driver/4.3.0/storage-class-parameters.json:35]: @mssachan I will make it more clear in upcoming PR

**Pull request merged**